### PR TITLE
Improvements for ImGui Hud Tuner

### PR DESCRIPTION
**Pull request by yohjimane**
Improved responsiveness by containing radio buttons and buttons within Tables, which resize dynamically based on current window width

Add some useful tooltips which display information when hovering over bone radio buttons and animation buttons, in case any of them clip due to text length.

### CODE REVIEW

**Comment by Xottab-DUTY** [comment]
So, is it ready for merge?

File: src/xrGame/player_hud_tune.cpp [hunk 1]
```diff
@@ -76,6 +71,12 @@ void CHudTuner::OnFrame()
     if (!g_player_hud)
         return;
 
+    auto CalcColumnCount = [](float columnWidth) -> int {
```
Comment (Xottab-DUTY) [inline src/xrGame/player_hud_tune.cpp]:
```suggestion
    auto calcColumnCount = [](float columnWidth) -> int
    {
```

File: src/xrGame/player_hud_tune.cpp [hunk 2]
```diff
@@ -187,11 +188,21 @@ void CHudTuner::OnFrame()
                 CDebugRenderer& render = Level().debug_renderer();
 
                 ImGui::SliderFloat("Debug Point Size", &debug_point_size, 0.00005f, 1.f, "%.5f");
-                if (ImGui::RadioButton("Draw Fire Point", draw_fp)) { draw_fp = !draw_fp; }; ImGui::SameLine();
-                if (ImGui::RadioButton("Draw Fire Point (GL)", draw_fp2)) { draw_fp2 = !draw_fp2; } ImGui::SameLine();
-                if (ImGui::RadioButton("Draw Fire Direction", draw_fd)) { draw_fd = !draw_fd; } ImGui::SameLine();
-                if (ImGui::RadioButton("Draw Fire Direction (GL)", draw_fd2)) { draw_fd2 = !draw_fd2; } ImGui::SameLine();
-                if (ImGui::RadioButton("Draw Shell Point", draw_sp)) { draw_sp = !draw_sp; }
+
+                if (ImGui::BeginTable("Show Debug Widgets", CalcColumnCount(210.f)))
```
Comment (Xottab-DUTY) [inline src/xrGame/player_hud_tune.cpp]:
```suggestion
                if (ImGui::BeginTable("Show Debug Widgets", calcColumnCount(210.f)))
```

File: src/xrGame/player_hud_tune.cpp [hunk 3]
```diff
@@ -257,31 +268,49 @@ void CHudTuner::OnFrame()
...
+
         if (current_hud_item && ImGui::CollapsingHeader("Bone and Animation Debugging", ImGuiTreeNodeFlags_DefaultOpen))
         {
             IKinematics* ik = current_hud_item->m_model;
             ImGui::Text("Bone Count = %i", ik->LL_BoneCount());
-            ImGui::Text("Root Bone = %s", ik->LL_BoneName_dbg(ik->LL_GetBoneRoot()));
+            ImGui::Text("Root Bone = %s, ID: %i", ik->LL_BoneName_dbg(ik->LL_GetBoneRoot()), ik->LL_GetBoneRoot());
 
-            for (const auto& [bone_name, bone_id] : *ik->LL_Bones())
+            if (ImGui::BeginTable("Bone Visibility", CalcColumnCount(125.f)))
```
Comment (Xottab-DUTY) [inline src/xrGame/player_hud_tune.cpp]:
```suggestion
            if (ImGui::BeginTable("Bone Visibility", calcColumnCount(125.f)))
```

File: src/xrGame/player_hud_tune.cpp [hunk 4]
```diff
@@ -257,31 +268,49 @@ void CHudTuner::OnFrame()
...
+                    }
+                    ik->LL_SetBoneVisible(bone_id, visible, FALSE);
+                }
+                ImGui::EndTable();
             }
 
             ImGui::NewLine();
-
-            for (const auto& [anim_name, motion] : current_hud_item->m_hand_motions.m_anims)
+            if (ImGui::BeginTable("Animations", CalcColumnCount(125.f)))
```
Comment (Xottab-DUTY) [inline src/xrGame/player_hud_tune.cpp]:
```suggestion
            if (ImGui::BeginTable("Animations", calcColumnCount(125.f)))
```

**Review by Xottab-DUTY — COMMENTED** [review]
Code style

**Review by Xottab-DUTY — APPROVED** [review]

**Comment by yohjimane** [comment]
@Xottab-DUTY code changes look good to me, ready for merge!

**Pull request merged**